tools/helmify/template/central-dashboard/centraldashboard-config.yaml

```diff
@@ -0,0 +1,130 @@
+apiVersion: v1
```

Comment:
why do we need this in helm templates since there are no changes to this configmap

Reply:
Fixed to get values for links.